lib/widgets/detail_views/contact_section.dart

```diff
@@ -23,8 +23,8 @@ class ContactSection extends StatelessWidget {
       child: Column(
         crossAxisAlignment: CrossAxisAlignment.start,
         children: [
-          Text(title, style: context.textTheme.headline),
-          const SizedBox(height: 16),
+          if (title != null) Text(title!, style: context.textTheme.headline),
```

Comment:
We don't want to use ! marks here.  Text should be just empty if null.

Reply:
Besides, this 2 conditions could be reduced to one.

Reply:
Actually, the text cannot be empty because it has some height, creating unnatural space on the widget